### DISCUSSION TITLE
call GiveError('Stress initialization from file not available for Double-point')

### DISCUSSION THREAD

**Discussion by EmreDumlu-UM** [Anura3D/Anura3D_OpenSource#233] [General]
Hello all,

I've recently working on a double-point method. I downloaded and compiled Anura3D_2024. Although I did not assign stresses from external file, I kept get the error 'Stress initialization from file not available for Double-point'. I've just looked at the related .FOR file then I suspected this part of the code may be written as reverse:

 if(NFORMULATION==2) then
     call GiveError('Stress initialization from file not available for Double-point')
     RETURN
 elseif (IsFollowUpPhase().or.(.not.CalParams%ApplyMapMPStressFromExtFile)) then
     RETURN
 end if

Then I shifted it as given below and compiled it again, the error disappeared and it ran.

 if(NFORMULATION==2) then
     if (IsFollowUpPhase() .or. (.not. CalParams%ApplyMapMPStressFromExtFile)) then
     RETURN
    else 
     call GiveError('Stress initialization from file not available for Double-point')
     RETURN
    end if    
 end if

I was curious, if anybody else further than me ever encountered this error in the double-point method.

Thanks in advance,
Emre

**Comment by 19Dario88** [comment]
Hi Emre,
could I ask you what version of Visual Studio and OneAPI you use to compile ANURA3D(2024)? With the same configuration when I compiled ANURA3D_2023, now several errors appear to me with the last version of the code.
Thank you in advance for your reply.
Dario

**Reply by EmreDumlu-UM** [reply]
Hi Dario. I used VS 2022 and HPC kit 2023 for Windows. I compiled it successfully by following the instructions given in the tutorial.

Best wishes

**Comment by pmarveggio** [comment]
@Emre-ncche thanks for your report and possible solution. We will discuss with my colleagues and fix the issue!

**Reply by AVroonland** [reply]
Hi @pmarveggio,
I wanted to check if you've had a chance to analyze this issue, as I'm encountering the same problem when trying to run a double-point model. Thank you in advance for your time and help!

**Comment by AVroonland** [comment]
Hi @Emre-ncche,
Could you please let me know which .FOR file you modified? Thank you in advance for your help!
